Services/CO.CDP.Organisation.WebApi/Model/Command.cs

```diff
@@ -336,12 +336,27 @@ public bool TryGetName(out string name)
     }
 }
 
+public record RegisterAuthenticationKey
+{
+    public required string Name { get; set; }
+    public required string Key { get; set; }
+    public Guid OrganisationId { get; set; }
+    public bool? Revoked { get; set; }
+}
+
+public record AuthenticationKey
+{
+    public required string Name { get; set; }
+    public bool? Revoked { get; set; }
+    public DateTimeOffset CreatedOn { get; set; }
+    public DateTimeOffset UpdatedOn { get; set; }
```

Comment:
As a rule of thumb CreatedOn/UpdatedOn properties should not be exposed to clients. Do we need them in the frontend? What for?

Reply:
This fields used to populate date created and date cancelled. 
![image](https://github.com/user-attachments/assets/ea4f9dcb-985c-495e-9f8e-a7a4d9afe32f)

Reply:
Done